hack/mkdocs-generate-conformance.py

```diff
@@ -50,28 +50,49 @@ def on_pre_build(config, **kwargs):
                                                 'HTTPRoutePathRedirect', 'HTTPRouteHostRewrite', 'HTTPRoutePathRewrite', 'HTTPRouteRequestMirror', 'HTTPRouteRequestMultipleMirrors', 'HTTPRouteRequestTimeout', 'HTTPRouteBackendTimeout', 'HTTPRouteParentRefPort']
 
 
+# NOTE add past versions of implementations to here when new release is cut
+past_versions = """
+??? info
+
+
+    The latest release of Gateway API is shown here. Past Gateway API implementations and feature support can be found here:
+
+    - [v1.0.0](./implementation-table-v1.0.0.md)
```

Comment:
This is probably a good opportunity to introduce an `implementation-tables` directory. That could also allow us to have working nav for the old reports. Currently https://deploy-preview-3088--kubernetes-sigs-gateway-api.netlify.app/implementation-table-v1.0.0/ loses nav because it's not present in mkdocs.yaml.

Reply:
good idea - i added it in!